src/site/xdoc/article2_4.xml

```diff
@@ -28,11 +28,11 @@ limitations under the License.
 issue to see what went on in the year of development between Lang 2.3 and 2.4.</p>
 <section name="Deprecations">
 <p>First, let us start with a couple of deprecations. As you can see in the release notes, we chose
-to deprecate the <a href="https://commons.apache.org/lang/api-release/org/apache/commons/lang/ObjectUtils.html#appendIdentityToString(java.lang.StringBuffer,%20java.lang.Object)"><code>ObjectUtils.appendIdentityToString(StringBuffer, Object)</code></a> method as its
+to deprecate the <a href="https://commons.apache.org/proper/commons-lang/javadocs/api-2.6/org/apache/commons/lang/ObjectUtils.html#appendIdentityToString(java.lang.StringBuffer,%20java.lang.Object)"><code>ObjectUtils.appendIdentityToString(StringBuffer, Object)</code></a> method as its
```

Comment:
I do not think it makes sense for docs for version 2.4 to point to 2.6 pages.

Reply:
You are right, that's fixed for both 2.4 and 2.5 changelogs.